src/prefect/runner/utils.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, Dict
+
+from prefect._vendor.fastapi import FastAPI
+from prefect._vendor.fastapi.openapi.utils import get_openapi
+
+from prefect import __version__ as PREFECT_VERSION
+
+
+def _inject_schemas_into_generated_openapi(
```

Comment:
Can you add some tests for the functions in this file? I'm having trouble tracking what they are doing, and I think some tests would help.